modules/acquia_cms_common/composer.json

```diff
@@ -100,11 +100,6 @@
             },
             "drupal/focal_point": {
                 "3162210 - Preview link accidentally closes the media library": "https://www.drupal.org/files/issues/2020-10-06/3162210-17.patch"
-            },
-            "drupal/checklistapi": {
```

Comment:
first patch is not available in 2.1.x, so we can not remove it.
cc @chandan-singh7929